app/use-cases/transfer_record.rb

```diff
@@ -17,4 +17,19 @@ def self.execute(record_id)
       end
     end
   end
+
+  def self.execute_organisation(record_id)
+    OrganisationExpressionOfInterest.transaction do
+      application = OrganisationExpressionOfInterest.find(record_id)
+      application.transferred_at = Time.zone.now
+      application.save!
+
+      uri = URI(ENV["REMOTE_API_URL"])
+      token = ENV["REMOTE_API_TOKEN"]
+      res = Net::HTTP.post(uri, application.answers.to_json, "Content-Type" => "application/json", "Authorization" => "Bearer #{token}")
```

Comment:
Realised this should just be `application.as_json`

Reply:
Also in the individual version

Reply:
Changed